app/javascript/react/components/MonsterShow.js

```diff
@@ -0,0 +1,60 @@
+import { Binding } from "@babel/traverse";
```

Comment:
It looks like this can be removed if "Binding" isn't being used anywhere in this file

Reply:
We don't know where it came from, but it's gone now.